README.md

```diff
@@ -222,6 +222,10 @@ __Note__: `SplashScreen` value should be absolute in order to work in a sub-page
 
         <preference name="SplashScreenBackgroundColor" value="0xFFFFFFFF"/>
 
+- Extended splashscreen on Windows does not require the plugin (as opposed to Android and iOS) in case you don't use the plugin API.  To disable the splashscreen add the following preference to config.xml: 
```

Comment:
We shouldn't be documenting this under quirks. Also does the disabling preference applies only to 'Windows' platform?